packages/components/.storybook/DocumentationTemplate.mdx

```diff
@@ -0,0 +1,22 @@
+import { ArgTypes, Canvas, Controls, Description, Markdown, Meta, Source, Stories, Title } from '@storybook/blocks';
+
+<div style={{marginLeft: 20, marginRight: 20}}>
```

Comment:
```suggestion
<div className="tw-mx-5">
```
And you need to add the `.mdx` to `content` in `tailwind.config.ts`

Reply:
changed

Reply:
Not pushed seems like

Reply:
now pushed

Reply:
a failed netlifybuild does not block the pipeline, this does